R/preprocess.R

```diff
@@ -94,6 +94,9 @@ enw_extend_date <- function(metaobs, max_delay = 20) {
 #' @export
 #' @importFrom data.table as.data.table copy
 enw_assign_group <- function(obs, by = c()) {
+  if ("group" %in% names(obs)) {
+    stop("Dataset cannot have a column called 'group'.")
+  }
```

Comment:
I just came across this potential bug if the dataset has a column `group`. Should we refactor to using a less frequent name?

Reply:
yes agree. Shall we make a new issue. Would be good to get this into `0.1.0` as another breaking change.

Reply:
Opened an issue for this (#99)